tpm2/structures.go

```diff
@@ -1023,8 +1025,57 @@ type AuthCommand struct {
 
 // TPMLDigest represents the TPML_Digest structure
 // It is used to convey a list of digest values.
-//This type is used in TPM2_PolicyOR() and in TPM2_PCR_Read()
+// This type is used in TPM2_PolicyOR() and in TPM2_PCR_Read()
 type TPMLDigest struct {
-	Count   uint32
 	Digests []tpmutil.U16Bytes
 }
+
+func (list *TPMLDigest) String() string {
+	var s strings.Builder
+	s.WriteString("TPMLDigest{")
```

Comment:
TPMLDigest and each Digest has an opening brace `{` - do we want to emit a closing brace `}` at the end as well to balance them out?

Reply:
fixed it

Reply:
I see an outer opening brace on line 1035, and then a pair of braces for each item (1038, 1040), and no outer closing brace - am I reading it correctly?